virttest/qemu_vm.py

```diff
@@ -225,6 +225,36 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
         self.last_boot_index = 0
         self.last_driver_index = 0
 
+    def _get_cmdline_format_cfg(self):
+        """
+        get data from file or input from parameter and then convert data to dict
+        :return: style data in dict
+        """
+        def _file(filepath):
+            if not filepath:
+                raise ValueError("The filepath is empty!")
+            if not os.path.isabs(filepath):
+                filepath = os.path.join(data_dir.get_data_dir(), filepath)
```

Comment:
I guess the conf file are saved and backend not data dir:  `data_dir.get_backend_cfg_path('qemu', filepath)`.